src/main/resources/changesets/changelog-consent-72.0.xml

```diff
@@ -0,0 +1,41 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="72.0" author="raejohanek">
+        <validCheckSum>ANY</validCheckSum>
+
+        <addColumn tableName="data_access_request">
+            <column name="collection_id" type="bigint"/>
+        </addColumn>
```

Comment:
We might consider a FK constraint on this to point to the `dar_collection` table.